frontend/encore/bootstrap.rst

```diff
@@ -1,18 +1,18 @@
 Using Bootstrap CSS & JS
-========================
+============================
```

Comment:
Should be reverted.

Reply:
Done in c90f20a94e2e46d9ba2f9cedeafc2845627414ba. The other issues were fixed in 6b732a65b3e9c716d214dffa7461922e6c5ca0c6. Thanks for reporting.